backend/src/v4/routes/config.js

```diff
@@ -140,6 +141,8 @@ function createClientConfig(serverConfig, req) {
 
 	clientConfig.impliedPermission = C.IMPLIED_PERM;
 
+	clientConfig.apryse = config.apryse;
```

Comment:
if i'm reading this right, the client config will have the following:

````
  {
      apryseLicense: "xxx" //from L60,
      apyrse: {
         licenseKey: "xxx" //this line
      }
  }
````

Do you need both?

Reply:
No that line is out of date 😮 Removed!